user-access-event-handlers/src/test/java/no/unit/nva/useraccess/events/service/UserMigrationServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package no.unit.nva.useraccess.events.service;
+
+import no.unit.nva.customer.model.CustomerDto;
+import no.unit.nva.customer.service.CustomerService;
+import no.unit.nva.useraccessmanagement.dao.UserDb;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.net.URI;
+
+import static java.util.UUID.randomUUID;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+public class UserMigrationServiceTest {
+
+    private static final URI SAMPLE_ORG_ID = URI.create("https://localhost/cristin/organization/123.0.0.0");
+
+    private CustomerService customerService;
+    private UserMigrationService userMigrationService;
+
+    @BeforeEach
+    public void init() {
+        customerService = Mockito.mock(CustomerService.class);
```

Comment:
I think it is better if we use a local dynamo client in this or another in PR instead of mocking the service. It would be much easier to follow the test scenarios.

Reply:
I think mocking is enough for our simple scenario. We can change this in the future if communication with CustomerService becomes more complex.